packages/gcf-utils/test/server.ts

```diff
@@ -168,3 +175,50 @@ describe('GCFBootstrapper', () => {
     });
   });
 });
+
+describe('getServer', () => {
+  let server: http.Server;
+  let handlerFunction: HandlerFunction;
+  before(done => {
+    nock.enableNetConnect(host => {
+      return host.startsWith('localhost:');
+    });
+    handlerFunction = async (
+      request: RequestWithRawBody,
+      response: express.Response
+    ) => {
+      if (request.rawBody) {
+        response.sendStatus(200);
+      } else {
+        response.sendStatus(400);
+      }
+    };
+    server = getServer(handlerFunction)
+      .on('listening', () => {
+        done();
+      })
+      .listen(TEST_SERVER_PORT);
+  });
+  after(done => {
+    server.on('close', () => {
+      done();
+    });
+    server.close();
+  });
+
+  it('should inject rawBody into the request', async () => {
+    const payload = require(resolve(fixturesPath, './issue_event'));
+    const response = await gaxios.request({
+      url: `http://localhost:${TEST_SERVER_PORT}/`,
+      headers: {
+        'x-github-delivery': '123',
+        'x-cloudtasks-taskname': 'test-bot',
+        'x-github-event': 'issues',
+        'content-type': 'application/json',
+      },
+      method: 'POST',
+      body: JSON.stringify(payload),
```

Comment:
Can you add indent here and verify the buffer retains the indent?

Reply:
Modified the test server to return the rawBody in the response and we assert that the body indentation has not changed.